src/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -557,6 +562,7 @@ public void stopGame() {
     m_showHistoryAction = null;
     m_showMapOnlyAction = null;
     m_showCommentLogAction = null;
+    m_showUnitNationAction = null;
```

Comment:
All components are set to null due to a bug in java 5 (1.5) Should I just remove since I don't think this bug iis still present?